crypto/x509/v3_addr.c

```diff
@@ -590,7 +590,12 @@ int X509v3_addr_add_prefix(IPAddrBlocks *addr,
 {
     IPAddressOrRanges *aors = make_prefix_or_range(addr, afi, safi);
     IPAddressOrRange *aor;
-    if (aors == NULL || !make_addressPrefix(&aor, a, prefixlen))
+    int maxprefixlen = length_from_afi(afi) * 8;
+
+    if (aors == NULL
+            || prefixlen < 1
```

Comment:
A prefix length of 0 is actually used by RPKI trust anchors for the special case of the entire IP address spaces `0.0.0.0/0` and `::/0`.

Here is an example: https://console.rpki-client.org/ta/ripe/ripe-ncc-ta.cer.html

Reply:
Ah! Right. Thanks. That makes sense. Fixed.